packages/runtime/src/index.ts

```diff
@@ -161,15 +162,15 @@ export default function init(inlineConfig: RuntimeOptions = {}) {
       styleElements.set(layer, styleElement)
 
       if (previousLayer == null) {
-        html().prepend(styleElement)
+        inject(styleElement)
       }
       else {
         const previousStyle = getStyleElement(previousLayer)
         const parentNode = previousStyle.parentNode
         if (parentNode)
           parentNode.insertBefore(styleElement, previousStyle.nextSibling)
```

Comment:
This should be handled. Maybe use `previousStyle`/`previousLayer` for context

Reply:
Not sure I undestand, I'm simply replacing the hardcoded `html()` with an optional custom method that prepends/appends the style element? What do you mean by "should be handled"? (I mean if a runtime user uses this config option he/she should know what it does IMHO 😉 )

Reply:
Ah maybe i get what you mean: You think we should give users the ability to inject each layer separately wherever they want it?